about/pricing-and-account-management.md

```diff
@@ -65,11 +65,13 @@ from initial development through to mission-critical enterprise applications.
 
 We offer new users a free, 30-day trial period of our $PERFORMANCE plan with no credit card required.  
 During your trial, you can contact sales@timescale.com to request information about, and access
-to, our $SCALE plan to determine how it fits your needs. Once you become a paying user, we can make 
-certain features of higher $PRICING_PLANs available to you for testing without upgrading.
+to, our $SCALE plan to determine how it fits your needs. During your trial, $SERVICE_SHORTs that receive no queries for 7 days are paused without any data loss. You easily resume them in $CONSOLE.
```

Comment:
think we could improve it like this:

```
If a $SERVICE_SHORT doesn’t receive any queries for 7 days, it will be paused to conserve resources. Your data remains intact during the trial, and you can easily resume the service from $CONSOLE. After your trial ends, we may remove your data unless you’ve added a payment method.
```